src/allencell_ml_segmenter/_tests/widgets/test_model_download_dialog.py

```diff
@@ -41,6 +44,17 @@ def model_download_dialog(
             "adding_headers": {"X-Foo": "Bar"},
         }
     )
+
+    responses.add(
+        **{
+            "method": responses.GET,
+            "url": f"{STG_BUCKET}?list-type=2",
```

Comment:
Rather that changing this on main, I think we should maintain a STG branch.  Thoughts?